python/ql/test/experimental/dataflow/regression/test.py

```diff
@@ -50,7 +50,7 @@ def sink3(cond, arg):
     if cond:
         sink(arg)
 
-def test8(cond):
+def test8(cond):  # This flow is shadowed by previous tests, perhaps do a path query
```

Comment:
Shadowing: Flow from line 10 to line 13 is already established by `test5`.
Broken: It looks like `source2` should simply not take an argument.

Reply:
> Shadowing: Flow from line 10 to line 13 is already established by test5.

I still don't understand what you mean by shadowing. Do you mean that the test is redundant since it is a duplicate of test5? I agree they are similar, but I won't say test8 is redundant: since this tests adds a layer of indirection (through an extra function) for flow to succeed.

> Broken: It looks like source2 should simply not take an argument.

yes, let's do that :+1: